src/callbacks_step/analysis.jl

```diff
@@ -315,7 +315,7 @@ function (analysis_callback::AnalysisCallback)(u_ode, du_ode, integrator, semi)
                     @sprintf("% 14d", nelementsglobal(mesh, solver, cache)))
 
         # Level information (only show for AMR)
-        print_amr_information(integrator.opts.callback, mesh, solver, cache)
+        print_level_information(integrator.opts.callback, mesh, solver, cache)
```

Comment:
In the line above (I cannot comment on that because it is not changed) the comment say "only show for AMR". This is not true anymore, is it?